flake.nix

```diff
@@ -30,7 +30,7 @@
             inherit inputs pkgs;
             modules = [
               ({ pkgs, lib, ... }: {
-                packages = [ pkgs.millet pkgs.polyml pkgs.smlfmt pkgs.gnumake pkgs.gcc pkgs.glibc ];
+                packages = [ pkgs.millet pkgs.mlton pkgs.polyml pkgs.smlfmt pkgs.gnumake pkgs.gcc pkgs.glibc pkgs.libressl ];
```

Comment:
mlton too? why?

Reply:
I don't remember, but I think it was because I wanted to use something on their basis

Reply:
Removed